src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -227,6 +238,11 @@ class CompilerContainer {
     this.compileTabLogic.runCompiler()
   }
 
+  onchangeLanguage (event) {
```

Comment:
Can it be "selectLangage" or "changeLangage". This is the action that is triggered on change from select.

Reply:
i'd keep it like that for the moment as the format `onSomething` is already used in this file